### ISSUE TITLE
Error when deleting last remaining language in Locale admin

### ISSUE THREAD

**Issue by Warqus** [wagtail/wagtail#6570]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

I get an error when trying to delete the last locale (English is the default language, probably not relevant):

![image](https://user-images.githubusercontent.com/14151318/99847300-2ca4cd80-2b78-11eb-8629-ef82fa3d8777.png)

![image](https://user-images.githubusercontent.com/14151318/99847325-33334500-2b78-11eb-99f8-6e5c98b788fb.png)

![image](https://user-images.githubusercontent.com/14151318/99847348-39c1bc80-2b78-11eb-99ca-16cc6d894681.png)

when clicking the 'Yes, delete', the error will occur.

### Steps to Reproduce

```
Internal Server Error: /admin/locales/2/delete/
Traceback (most recent call last):
  File "C:\Work\virtual_env\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "locale_id" violates not-null constraint
DETAIL:  Failing row contains (1, 0001, 1, 0, Root, root, t, f, /, , f, , null, null, f, 1, null, f, null, null, null, null, Root, null, null, 2ac624bf-c27c-4f38-8328-55abe117d0dc, null, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Work\virtual_env\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Work\virtual_env\lib\site-packages\django\core\handlers\base.py", line 179, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Work\virtual_env\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\wagtail\admin\urls\__init__.py", line 127, in wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\wagtail\admin\auth.py", line 193, in decorated_view
    return view_func(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\wagtail\admin\views\generic.py", line 45, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\django\views\generic\edit.py", line 218, in post
    return self.delete(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\wagtail_localize\locales\views.py", line 172, in delete
    return super().delete(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\wagtail\admin\views\generic.py", line 242, in delete
    response = super().delete(request, *args, **kwargs)
  File "C:\Work\virtual_env\lib\site-packages\django\views\generic\edit.py", line 213, in delete
    self.object.delete()
  File "C:\Users\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 74, in inner
    return func(*args, **kwds)
  File "C:\Work\virtual_env\lib\site-packages\wagtail\core\models.py", line 392, in delete
    root_page_with_this_locale.update(locale=new_locale)
  File "C:\Work\virtual_env\lib\site-packages\django\db\models\query.py", line 784, in update
    rows = query.get_compiler(self.db).execute_sql(CURSOR)
  File "C:\Work\virtual_env\lib\site-packages\django\db\models\sql\compiler.py", line 1522, in execute_sql
    cursor = super().execute_sql(result_type)
  File "C:\Work\virtual_env\lib\site-packages\django\db\models\sql\compiler.py", line 1156, in execute_sql
    cursor.execute(sql, params)
  File "C:\Work\virtual_env\lib\site-packages\django\db\backends\utils.py", line 98, in execute
    return super().execute(sql, params)
  File "C:\Work\virtual_env\lib\site-packages\django\db\backends\utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "C:\Work\virtual_env\lib\site-packages\django\db\backends\utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Work\virtual_env\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Work\virtual_env\lib\site-packages\django\db\utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Work\virtual_env\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "locale_id" violates not-null constraint
DETAIL:  Failing row contains (1, 0001, 1, 0, Root, root, t, f, /, , f, , null, null, f, 1, null, f, null, null, null, null, Root, null, null, 2ac624bf-c27c-4f38-8328-55abe117d0dc, null, null).
```

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes

**Comment by gasman** [comment]
Hi @Warqus - did you test this on a project without wagtail-localize installed? (For projects without wagtail-localize, this should have already been fixed in #6553.)

**Comment by Warqus** [comment]
@gasman Thanks for your reaction. I see it is indeed only occurring when using wagtail-localize. I will report it at the wagtail-localize repository.
